src/templates/kit/screens/DotVotingScreen.js

```diff
@@ -185,11 +185,11 @@ function DotVotingScreen({
         label={
           <React.Fragment>
             Minimum Participation %
-            <Help hint="What’s the minimum participation?">
+            <Help hint="What is Minimum Participation?">
               <strong>Minimum Participation</strong> is the minimum percentage
               of the total token supply that is required to participate in a dot
               vote for the proposal to be considered valid. For example, if
-              "Minimum Participation %" is set to 51%, then at least 51% of the
+              "Minimum Participation %" is set to 50%, then at least 50% of the
```

Comment:
Just to confirm: Dot Voting does in fact use "at least" for their threshold rather than "greater than", which the Voting app uses?

Reply:
They use the "same" logic as our Voting—100% is impossible.

Reply:
Discussed via DM. Copy has been updated in separate comments.